src/libraries/Microsoft.Extensions.FileProviders.Physical/tests/PhysicalFileProviderTests.cs

```diff
@@ -87,6 +89,48 @@ public void GetFileInfoReturnsNotFoundFileInfoForIllegalPathWithLeadingSlashes_U
             GetFileInfoReturnsNotFoundFileInfoForIllegalPathWithLeadingSlashes(path);
         }
 
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Linux)] 
+        public async void PollingFileProviderShouldntConsumeINotifyInstances()
+        {
+            List<IDisposable> disposables = new List<IDisposable>();
+            using var root = new DisposableFileSystem();
+
+            int callbacks = 0;
+            int maxInstances = int.Parse(File.ReadAllText("/proc/sys/fs/inotify/max_user_instances"));
```

Comment:
Is it safe to assume that this file exists in this path for all distros?

Reply:
Oh, I just noticed that the test was originally topped with `TestPlatforms.AnyUnix`. Since `Linux` is a more specific platform, I assume we can confidently expect the file to exist in the test environment for that platform.

Reply:
I still went ahead and added an assertion for the file existence to ensure we fail gracefully when absent.